src/static/css/index.css

```diff
@@ -422,3 +452,257 @@ header {
 	width: 25px;
 	opacity: 0.3;
 }
+
+/* Mobile View */
+@media (max-width: 1440px) {
```

Comment:
Let's use a 600px breakpoint for mobile. I'm also noticing that smaller desktop viewport sizes appear to have overflowing images:

![image](https://user-images.githubusercontent.com/1120896/65060754-76779f80-d946-11e9-9fe1-6fdd086c2451.png)

Reply:
> I'm also noticing that smaller desktop viewport sizes appear to have overflowing images

Yup, that's because I used a fixed height and width for these images currently. 

I think the suggestion to flatten the browser graphic with the three characters into a single image would be great so that responsive images can be accounted more easily here.